src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/induction/QualificationsAndTrainingResourceMapper.kt

```diff
@@ -94,17 +94,39 @@ abstract class QualificationsAndTrainingResourceMapper {
 
   fun toOffsetDateTime(instant: Instant?): OffsetDateTime? = instant?.atOffset(ZoneOffset.UTC)
 
+  fun toUpdatePreviousQualificationsDto(
+    request: UpdateEducationAndQualificationsRequest?,
+    prisonId: String,
+  ): UpdatePreviousQualificationsDto? {
+    return if (request == null) {
+      null
+    } else {
+      convertToUpdatePreviousQualificationsDto(request, prisonId)
+    }
+  }
+
   @Mapping(target = "reference", source = "request.id")
   @Mapping(target = "educationLevel", source = "request.educationLevel", defaultValue = "NOT_SURE")
-  abstract fun toUpdatePreviousQualificationsDto(
-    request: UpdateEducationAndQualificationsRequest,
+  abstract fun convertToUpdatePreviousQualificationsDto(
+    request: UpdateEducationAndQualificationsRequest?,
     prisonId: String,
   ): UpdatePreviousQualificationsDto?
 
+  fun toUpdatePreviousTrainingDto(
```

Comment:
It's currently not possible to create an Induction without providing a value for training (even if it's just `TrainingType.NONE`), but there's no reason to apply the same rule for an update.